.github/workflows/latest_docker_release.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish latest docker image
+
+on:
+  push:
+
+jobs:
+  build-gh:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: whoan/docker-build-with-cache-action@master
+      with:
+        username: ${{ github.actor }}
+        password: "${{ secrets.GITHUB_TOKEN }}"
+        image_name: jtcoolen/bot/coqbot
+        image_tag: latest
```

Comment:
Unfortunately, from what I understood from [here](https://github.community/t/overwrite-docker-image-in-github-package-registry/16433/10), because GitHub Packages doesn't allow deletion of public packages, the use case of pushing and overwriting the latest tag is not supported.

Reply:
Thanks for mentioning it. Alternatively, we could push to an other registry like Docker Hub from which we can overwrite tagged images ([there's even an action to do that](https://github.com/actions-hub/docker).)